lib/tasks/algolia.rake

```diff
@@ -3,8 +3,8 @@
 namespace :algolia do
   task reindex: :environment do
     print "Reindexing resource/service index... "
-    Resource.reindex!
-    Service.reindex!
+    Resource.where(status: :approved).reindex
```

Comment:
With the `.where`, can this become null? (looking at testerrors)

Reply:
Looking at the Travis errors, it looks like it's actually erroring on the `remove_from_index!` calls in the other lines of the diff.

Reply:
Ah ok, I'll take a deeper look.